packages/reactotron-app/App/Lib/api-to-curl.js

```diff
@@ -0,0 +1,30 @@
+export function apiRequestToCurl (payload = {}) {
+  const output = []
+  const request = payload.request || {}
+  const { method, headers, data, url } = request
+
+  if (method === "GET") {
+    output.push("curl")
+  } else {
+    output.push(`curl -X ${method} `)
+  }
+
+  for (let header in headers) {
+    output.push(` -H "${header}:${headers[header]}"`)
+  }
+
+  output.push(` ${url}`)
+
+  if (data) {
+    let parsedData = data
+    try {
+      if (typeof data === "string") {
+        parsedData = JSON.parse(parsedData)
+      }
+      parsedData = JSON.stringify(parsedData)
+    } catch (e) {}
+    output.push(` -d '${parsedData}'`)
+  }
+
+  return output.join("")
```

Comment:
I wonder if we'll need to do shell escaping?  For example, payloads will be in `json` which have `"` characters out the wazoo.  Should we escape them to be safe for the shell?

Reply:
Are ” safe when they’re wrapped in ‘ ?

Reply:
I don't think so... i think it'll probably end look like this:

```
curl -d "{\"lulz\": \"you won't escape\"}"
```

or 

```
curl -d '{"lulz": "you won\'t escape"}'
```

i think?  Not entirely positive on the shell escaping rules.

Reply:
Apparently one could also just escape everything that is not alphanumeric, or the boundary quotes.